src/top/utop.el

```diff
@@ -355,51 +360,6 @@ it is started."
       (utop-send-string (concat "data:" line "\n")))
     (utop-send-string "end:\n")))
 
-(defun utop-last-type ()
```

Comment:
Supposedly, this would b e useful to insert inferred type annotations from the toplevel into the source. However, there's now better ways to do this (merlin, lsp)